docs/content/tutorial/customize-cluster.md

```diff
@@ -198,6 +198,88 @@ spec:
 This volume can be removed later by removing the `walVolumeClaimSpec` section from the instance. Note that when changing the WAL directory, care is taken so as not to lose any WAL files. PGO only
 deletes the PVC once there are no longer any WAL files on the previously configured volume.
 
+## Custom Sidecar Containers for PostgreSQL Instance Pods
+
+PGO allows you to configure custom
+[sidecar Containers](https://kubernetes.io/docs/concepts/workloads/pods/#how-pods-manage-multiple-containers)
+for any of your PostgreSQL instance Pods.
+
+To use this feature, currently in `Alpha`, you will need to enable it via the PGO
+[feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/).
```

Comment:
Should this link go to our feature gate docs? Then there we could link off to Kubernetes feature gate docs

Reply:
I wasn't sure about linking to the README file as that's more for development guidance. I was thinking the Kubernetes documentation would be sufficient to make sure we explained the concept. :thinking: Have we linked to READMEs previously?

Reply:
Oh no I don't think we need to do that. Linking to the Kube docs is fine with me.